addons/event/static/src/views/event_registration_list_controller.js

```diff
@@ -0,0 +1,57 @@
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { onMounted, onWillUnmount } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { listView } from "@web/views/list/list_view";
+import { ListController } from "@web/views/list/list_controller";
+
+export default class EventRegistrationListController extends ListController {
+
+    setup() {
+        super.setup();
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+
+        onMounted(this.hideSystray);
+        onWillUnmount(this.showSystray);
+    }
+
+    async openRecord(record, mode) {
+        if (!this.props.context.name_with_seats_availability) {
```

Comment:
How is this linked to the feature? :)

Reply:
I was using the already present context.. :sweat_smile: but now I am adding a new one. (Also, by doing this it'll not break in the report and anywhere else) :smile: